ServerRunner/ServerRunner.cs

```diff
@@ -10,31 +10,27 @@ public class ServerRunner
     {
         public static void Main()
         {
-            Console.WriteLine("Path to DLL containing the distributed task:");
-            var distributedTaskAssemblyPath = Console.ReadLine();
-            var distributedTaskAssembly = Assembly.LoadFrom(distributedTaskAssemblyPath);
-            var distributedTask = GetTypeFromAssembly<IDistributedTask>(distributedTaskAssembly);
-
             /**
-             * The DLL with the ITask is loaded here just in order to run the task and for the IDistributedTask
+             * The DLL with the ISubtask and ITask is loaded here just in order to run the task and for the ITask
              * to be able to aggregate the results.
              * 
              * In the final version the input data will be sent to the distributed nodes who will be responsible
-             * for executing the ITask compiled to JS.
+             * for executing the ISubtask compiled to JS.
              */
             Console.WriteLine("Path to DLL containing the task:");
             var taskAssemblyPath = Console.ReadLine();
             var taskAssembly = Assembly.LoadFrom(taskAssemblyPath);
             var task = GetTypeFromAssembly<ITask>(taskAssembly);
+            var subtask = GetTypeFromAssembly<ISubtask>(taskAssembly);
 
-            Console.WriteLine($"Input data for the distributed task ({distributedTask.GetType().FullName}):");
+            Console.WriteLine($"Input data for the distributed task ({subtask.GetType().FullName}):");
```

Comment:
I reckon we should print the `task` name instead of the `subtask` name because that is the class that handles that data